app/views/entities/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<main class="body-container">
+<section class="container">
+    <% total_amount = 0 %>
+    <% @entities.each do |entity| %>
+          <% total_amount += entity.amount %>
+    <% end %>
+    <table>
+      <thead>
+      <tr>
+          <th> </th>
+          <th>Total Amount</th>
```

Comment:
- [ ] Categories show page should follow this design 
![image](https://github.com/ivanmvh/rails_wallet/assets/62705947/6e502380-b8e6-44e8-9a04-5452046375cd)

Reply:
There is not a show page for a category. There is a list of transactions (no one but the most recent transactions up) . I will adapt the list to this design. 
Here the requirement:
**Home page (categories page**)
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category. 

**Transactions page**
    For a given category, the list of transactions is presented, ordered by the most recent.
    At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
    There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
    When the user clicks on the "Back" button (<), the user navigates to the home page.